docker-for-windows/index.md

```diff
@@ -248,30 +243,11 @@ username and password. Enter your credentials and then click **Apply & Restart**
 >
  * Docker Desktop sets permissions to read/write/execute for users, groups and others [0777 or a+rwx](http://permissions-calculator.org/decode/0777/).
    This is not configurable. See [Permissions errors on data directories for shared volumes](troubleshoot.md#permissions-errors-on-data-directories-for-shared-volumes).
->
- * Ensure the domain user has access to shared drives, as described in [Verify domain user has permissions for shared drives](troubleshoot.md#verify-domain-user-has-permissions-for-shared-drives-volumes).
```

Comment:
Do we know what share / number of users are still on older versions? Thinking if we need to move the content to a separate page, and add a mention "if you're using version XX or older", which either instructs them to upgrade to the latest & greatest, or point to the troubleshooting page for older versions.


/cc @usha-mandya @stephen-turner @nebuk89  for thoughts

Reply:
I believe we support our latest and greatest on Docker Desktop. Therefore, not sure if moving obsolete features to another page and duplicating the entire topic is a good option.  Users can browse through the release notes to identify features added/removed from specific versions.

Reply:
I agree with Usha. We can't keep documenting all old versions when we change something.

Reply:
Okay, thanks for checking. I've seen this error/situation popping up in many situations (some users linger a bit behind "latest & greatest"), so wanted to double check 👍